praw/models/reddit/subreddit.py

```diff
@@ -1447,6 +1457,11 @@ def update(
             valid emoji string, for example ``':snoo:'``.
         :param max_emojis: (int) Maximum emojis in the flair
             (Reddit defaults this value to 10).
+        :param fetch: Whether or not PRAW will fetch existing information on
+            the existing flair before updating (Default: False).
+
+        .. warning:: ``False``, all parameters not provided will be reset to
+            default (``None`` or ``False``) values.
```

Comment:
I think part of this sentence is missing.

Reply:
It seems to be.